Sources/Components/Fiks Ferdig/FiksFerdigInfoView/FiksFerdigShippingInfoViewModel.swift

```diff
@@ -3,17 +3,20 @@ import UIKit
 public class FiksFerdigShippingInfoViewModel {
     public let providers: [FiksFerdigShippingInfoCellViewModel]
     public let headerViewModel: FiksFerdigAccordionViewModel
+    public let noProviderText: String
 
     public init(
         headerTitle: String,
         providers: [FiksFerdigShippingInfoCellViewModel],
+        noProviderText: String,
```

Comment:
Btw, this is a breaking change since you're adding a new parameter without a default value.